src/main/java/com/github/tomakehurst/wiremock/common/Json.java

```diff
@@ -50,6 +51,7 @@ protected ObjectMapper initialValue() {
           objectMapper.configure(JsonParser.Feature.IGNORE_UNDEFINED, true);
           objectMapper.configure(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN, true);
           objectMapper.configure(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS, true);
+          objectMapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS, true);
```

Comment:
This is `@Deprecated` it recommends using `JsonMapper` instead of `ObjectMapper` when supplying `MapperFeature`s.